ynr/apps/candidates/models/popolo_extra.py

```diff
@@ -131,6 +155,72 @@ def has_results(self):
             return True
         return False
 
+    @property
+    def polls_closed(self):
+        # TODO: City of London and other complex cases. Take this from EE?
+        normal_poll_close_time = datetime.time(hour=22)
+
+        poll_close_datetime = timezone.make_aware(
+            datetime.datetime.combine(
+                self.election.election_date, normal_poll_close_time
+            )
+        )
+        return poll_close_datetime <= timezone.now()
+
+    @property
+    def has_lock_suggestion(self):
+        return self.suggestedpostlock_set.exists()
+
+    @property
+    def get_winner_count(self):
+        """
+        Returns 0 rather than None if the winner_count is unknown. See comment in
+        https://github.com/DemocracyClub/yournextrepresentative/pull/621#issuecomment-417252565
+
+        :return:
+        """
+        if self.winner_count:
+            return self.winner_count
+        return 0
+
+    def user_can_edit_membership(self, user):
+        """
+        Can a given user edit this ballot?
+
+        """
+
+        # users have to be logged in to an account
+        if not user.is_authenticated:
+            return False
+
+        # If the ballot is unlocked, anyone can edit the memberships
+        if not self.candidates_locked:
+            return True
+
+        # If a user is trusted to lock then they can edit memberships
+        # TODO: Is this right?
+        # https://github.com/DemocracyClub/yournextrepresentative/issues/991
+        if user.groups.filter(name=TRUSTED_TO_LOCK_GROUP_NAME).exists():
+            return True
+
```

Comment:
Also, can we terminate this method with an explicit `return False` instead of relying on implicit `return None`